add configs and js files

kozak-iz-kh opened the pull request.

ihor4568 commented on dist/js/main.js:
```diff
@@ -0,0 +1,125 @@
+/******/ (function(modules) { // webpackBootstrap
```
please do not commit files that are result of the build

ihor4568 commented:
1 - Why there are no reviewers in this PR?
3 - Extra task: please try to use https://github.com/jantimon/html-webpack-plugin instead of creating html manually.

kozak-iz-kh commented:
> 1 - Why there are no reviewers in this PR?
> 3 - Extra task: please try to use https://github.com/jantimon/html-webpack-plugin instead of creating html manually.

1. added reviewers.
2. deleted bundle files
3. extra task completed
![screenshot_7](https://user-images.githubusercontent.com/37671111/46019859-b0bcb100-c0e5-11e8-9674-2bbc4cd3e06f.png)
